### PR TITLE
Update manage-tags.md

### PR DESCRIPTION
**Pull request by AlexanderHolmeset**
Org-wide settings are moved to Teams settings.
Screenshot under Manage custom tags for your org..... is outdated,

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-tags.md [hunk 1]
```diff
@@ -67,12 +67,13 @@ By default, team owners can add custom tags. You can change this setting to allo
...
-    - **Suggested default tags**: Use this to add a set of default tags. You can add up to 25 tags, and each tag can contain a maximum of 25 characters. Team owners and members (if the feature is enabled for them) can use these suggestions, add to them, or create a new set of tags.
+    - **Suggested tags**: Use this to add a set of default tags. You can add up to 25 tags, and each tag can contain a maximum of 25 characters. Team owners and members (if the feature is enabled for them) can use these suggestions, add to them, or create a new set of tags.
     - **Let custom tags be created**: Turn on this setting to let people add tags other than the suggested default tags that you set. If this is turned off, people can only use the suggested default tags. If you turn this off, make sure that you add one or more default tags.
+    - **Allow Shifts app to apply tags**: Turn on this setting to whether or not to enable Tagging by Shift.
```
Comment (JohanFreelancer9) [inline Teams/manage-tags.md:76]:
```suggestion
    - **Allow Shifts app to apply tags**: Turn on this setting to enable Tagging by Shift.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AlexanderHolmeset, kindly commit the suggested change if you agree with it. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome Hope all is well. Wondering if you can help with this request? Thanks!

**Pull request closed without merging**